tests/k8s/1-001_validate_basic/01-assert.yaml

```diff
@@ -52,7 +52,7 @@ metadata:
 kind: ServiceAccount
 apiVersion: v1
 metadata:
-  name: example-argocd-argocd-redis-ha
+  name: example-argocd-argocd-redis
```

Comment:
Since the naming convention has changed in the current code, we can update all the names of service account and rolebindings in this test file to skip `argocd` duplication, 
For example - `name: example-argocd-redis`

Reply:
updated